lib/experimental/assets/tinymce-proxy.js

```diff
@@ -0,0 +1,14 @@
+async function markAsRequiringTinymce() {
+	const currentUrl = new URL( window.location.href );
+	currentUrl.searchParams.set( 'requiresTinymce', '1' );
```

Comment:
We're using a query argument in order to be able to detect we need TinyMCE from the server.